testing/environments/snapshot.yml

```diff
@@ -21,4 +25,6 @@ services:
     build:
       context: ./docker/kibana
       dockerfile: Dockerfile-snapshot
-
+      args: [ ]
+        #KIBANA_DOWNLOAD_URL: https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha1-SNAPSHOT-linux-x86_64.tar.gz
```

Comment:
So we don't need them at all? Or do we have to uncomment this in some situation?